distributions/otelcol-otlp/manifest.yaml

```diff
@@ -6,12 +6,26 @@ dist:
   output_path: ./_build
 
 exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.125.0
   - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.125.0
   - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.125.0
 
+
+extensions:
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.125.0
+
+
 receivers:
   - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.125.0
 
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.125.0
```

Comment:
We should definitely not add this one, we are still discussing whether this is the right call for this distribution or not